Template/Tag/EtrackerTag.web.js

```diff
@@ -0,0 +1,53 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+		var etrackerConfig = parameters.get('etrackerConfig', {});
+		var trackingType = parameters.get('trackingType');
+		if (trackingType === 'pageview') {
+			var etrackerID = etrackerConfig.etrackerID;
+            		if (etrackerID) {
+				// Pageview parameters for overwrite
+				if(etrackerConfig.et_pagename){

```

Comment:
random thought...  does maybe this make sense?

```js
for (i in etrackerConfig) {
   if (TagManager.utils.hasProperty(etrackerConfig, i)) {
      window[i] = etrackerConfig[i];
  }
}
```

This way you might be able to get rid of all the `if`s below